server/src/main/java/org/uiautomation/ios/utils/SimulatorSettings.java

```diff
@@ -285,47 +286,92 @@ public void resetContentAndSettings() {
         System.err.println("couldn't re-create: " + contentAndSettingsFolder);
       }
     } else {
-      // Starting with Xcode 6 and later, we can use simctl to do the hard work for us.
-      // If it fails, we have to turn the device off first.
       if (!eraseSimulator()) {
-        log.log(Level.WARNING, "Reset content and settings failed, " +
-            "possibly device is in booted state, shutdown device = " + deviceUUID);
-        List<String> simctlArgs = new ArrayList<>();
-        simctlArgs = Arrays.asList("xcrun", "simctl", "shutdown", deviceUUID);
-        Command simctlCmd = new Command(simctlArgs, true);
+        log.info("Erase contents and settings failed on this device: " + deviceUUID);
```

Comment:
are you saying that warnings would have come from `eraseSimulator()` and therefore it's not that useful to issue another warning?  if so, it probably makes sense to change this message so that it's stating what is about to happen (instead of describing the failure that just happened).